File: scratch/fix_css.py
import sys

file_path = r'c:\Users\Paulo Prado\Downloads\Compressed\ecossistema\src\pages\MitosisVisualization.css'

with open(file_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# Target line 307 (1-indexed is 306 in 0-indexed)
# Wait, let's look for the point where it broke.
# Line 306 is "justify-content: center;"
# Line 307 is "background: #081a10;"

insertion_point = -1
for i, line in enumerate(lines):
    if 'justify-content: center;' in line and i > 300:
        insertion_point = i + 1
        break

if insertion_point != -1:
    new_content = [
        '  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.18);\n',
        '  overflow: hidden;\n',
        '  z-index: 1;\n',
        '}\n',
        '\n',
        '.mv-appcircle-img {\n',
        '  width: 48px;\n',
        '  height: 48px;\n',
        '  object-fit: contain;\n',
        '}\n',
        '\n',
        '/* ─── Cell cores ──────────────────────────────────────────────────────────── */\n',
        '.mv-core {\n',
        '  position: absolute;\n',
        '  border-radius: 50%;\n',
        '  display: flex;\n',
        '  flex-direction: column;\n',
        '  align-items: center;\n',
        '  justify-content: center;\n',
        '  gap: 5px;\n',
        '  z-index: 5;\n',
        '}\n',
        '\n'
    ]
    lines[insertion_point:insertion_point] = new_content
    with open(file_path, 'w', encoding='utf-8') as f:
        f.writelines(lines)
    print("Success")
else:
    print("Insertion point not found")

File: scratch/fix_css.js
const fs = require('fs');
const path = require('path');

const filePath = 'c:\\Users\\Paulo Prado\\Downloads\\Compressed\\ecossistema\\src\\pages\\MitosisVisualization.css';
let content = fs.readFileSync(filePath, 'utf8');

const targetPart = `  display: flex;
  align-items: center;

.mv-core--care {
  background: transparent;
  border: none;
  box-shadow: none;
}`;

const replacementPart = `  display: flex;
  align-items: center;
  justify-content: center;
  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.18);
  overflow: hidden;
  z-index: 1;
}

.mv-appcircle-img {
  width: 48px;
  height: 48px;
  object-fit: contain;
}

/* ─── Cell cores ──────────────────────────────────────────────────────────── */
.mv-core {
  position: absolute;
  border-radius: 50%;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  gap: 5px;
  z-index: 5;
}

.mv-core--grupo {
  background: #081a10;
  border: 3px solid #a1ce28;
  box-shadow: 0 0 0 6px rgba(161, 206, 40, 0.15), 0 0 36px rgba(161, 206, 40, 0.4);
}

.mv-core--care {
  background: transparent;
  border: none;
  box-shadow: none;
}`;

if (content.includes(targetPart)) {
    content = content.replace(targetPart, replacementPart);
    fs.writeFileSync(filePath, content, 'utf8');
    console.log('Success');
} else {
    console.log('Target part not found');
}
